ext/enchant/enchant.stub.php

```diff
@@ -40,15 +40,15 @@ final class EnchantDictionary
 
 function enchant_broker_init(): EnchantBroker|false {}
 
-/** @deprecated */
+#[\Deprecated]
 function enchant_broker_free(EnchantBroker $broker): bool {}
 
 function enchant_broker_get_error(EnchantBroker $broker): string|false {}
 
-/** @deprecated */
+#[\Deprecated]
 function enchant_broker_set_dict_path(EnchantBroker $broker, int $type, string $path): bool {}
 
-/** @deprecated */
+#[\Deprecated]
 function enchant_broker_get_dict_path(EnchantBroker $broker, int $type): string|false {}
```

Comment:
Those are also deprecated in 8.1, seemingly in favour of the OO API, @kocsismate do you remember more what was happenning around this time?

Reply:
@Girgias I don't, but I managed to find it: https://github.com/php/php-src/pull/5577#issuecomment-635967011